data/product.attribute.csv

```diff
@@ -1,14 +1,8 @@
 id,name,create_variant
-product_attribute_BbA,Baby by Age,TRUE
 product_attribute_BS,Bra Sizes,TRUE
-product_attribute_CbA,Children by Age,TRUE
 product_attribute_DS,Diaper Sizes,TRUE
-product_attribute_MS,Mixed Sizes,FALSE
-product_attribute_O,Onesize,FALSE
 product_attribute_CS,Clothing Sizes,TRUE
-product_attribute_CSS,Children Shoe Sizes,TRUE
-product_attribute_FSS,Female Shoe Sizes,TRUE
-product_attribute_MSS,Male Shoe Sizes,TRUE
+product_attribute_SS, Shoe Sizes,TRUE
```

Comment:
Not sure if it matters for actual import, but I'm guessing the space here is a typo?